js/pages/incidence-rates/ir-manager.html

```diff
@@ -54,6 +56,89 @@
 					<concept-set-builder params="conceptSets: ConceptSets, ref: $parent.conceptSetEditor"></concept-set-builder>
 				<!-- /ko -->			
 			</div>
+			<div role="tabpanel" data-bind="css: { active: $component.activeTab() == 'export' }" class="tab-pane">
+				<ul class="nav nav-pills">
+					<li role="presentation" data-bind="css: { active: $component.exportTabMode() == 'sql' }, click: function() { $component.activeTab('sql');}">
+						<a>SQL</a>
+					</li>
+				</ul>
+
+				<div class="tab-content">
+					<div role="tabpanel" data-bind="css: { active: $component.exportTabMode() == 'sql'}" class="tab-pane">
+						<div class="row">
```

Comment:
This is copy-paste from Cohort Definitions, isn't it? Let's move this to a shared component and not introduce code duplication